codegen/generator/ts/templates/functions_test.go

```diff
@@ -0,0 +1,70 @@
+package templates
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFormatName(t *testing.T) {
+	cases := map[string]struct {
+		in   string
+		want string
+	}{
+		"fooId -> fooId":           {"fooId", "fooId"},
+		"containers -> containers": {"containers", "containers"},
```

Comment:
So when there must be any transformation?

Reply:
it will be cleaned up if it needs at the end.

Reply:
I will keep it in until the release. In case I need that for further formatting.